src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedIntrinsicOperatorSymbol.cs

```diff
@@ -163,6 +163,11 @@ public override DllImportData GetDllImportData()
             return null;
         }
 
+        public override bool AreLocalsZeroed
+        {
+            get { throw ExceptionUtilities.Unreachable; }
```

Comment:
Do we not generate any code for this symbol?

Reply:
@agocke's comment below is a good answer to this question.

Reply:
Looks like it's only used in the semantic model, so I suspect not.